ckanext/s3filestore/theme/templates/s3filestore/snippets/direct_upload_module.html

```diff
@@ -0,0 +1,12 @@
+<div
+    data-module="s3filestore-direct-upload"
+    data-module-bucket="{{ h.s3filestore_get_bucket }}"
+    data-module-aws_key="{{ h.s3filestore_get_aws_key }}"
```

Comment:
I see we're using the AWS key here -- is this required or can we use an AWS role if we're on a AWS hosted machine?